src/ng/animateRunner.js

```diff
@@ -85,7 +85,7 @@ var $$AnimateRunnerFactoryProvider = function() {
 
         // the document may not be ready or attached
         // to the module for some internal tests
-        if (doc && doc.hidden) {
+        if ($$isDocumentHidden()) {
```

Comment:
Ah, I confused this with the `$$rAF`'s tick. Since it's indeed called one, I guess it's fine as it is.